util/find-doc-nits

```diff
@@ -601,6 +601,13 @@ sub wording {
     }
     err($id, "found 'epoch' should use 'Epoch'")
         if $contents =~ /\bepoch\b/;
+    if ( $id =~ m@man1/@ ) {
+        err($id, "found 'tool' in NAME, should use 'command'")
+            if $contents =~ /=head1 NAME.*tool.*=head1 SYNOPSIS/s;
+        err($id, "found 'utility' in NAME, should use 'command'")
+            if $contents =~ /NAME.*tool.*=head1 SYNOPSIS/m;
```

Comment:
This is matching for tool and not utility also `/m` modifier does not look correct.

Reply:
yeah, incomplete edit.  fixup commit pushed, thanks!